delegation-toolkit/get-started/delegation-quickstart.md

```diff
@@ -146,7 +146,7 @@ const userOperationHash = await bundlerClient.sendUserOperation({
   account: delegateSmartAccount,
   calls: [
     {
-      to: delegateSmartAccount.address,
+      to: delegatorSmartAccount.address,
```

Comment:
I'm not actually sure if we should document delegatorSmartAccount. It assumes the delegatorSmartAccount will be present always which is not the case, if the delegations are stored, or if delegations are used in different environment to redeem like server, it won't stand true. Hence it was using delegateSmartAccount.address. Maybe we should change the documentation for redeemDelegation rather than quickstart. WDYT?

Reply:
If we keep `delegateSmartAccount` here, what about the issue stated in #2082?
> Using the delegate's address causes the `UserOperation` to fail, often with no clear error unless debugged deeply. The delegation signature isn't validated on the delegate’s account, which leads to invalid operations.